Assets/MRTK/SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs

```diff
@@ -183,6 +183,10 @@ private void OnEnable()
             // Begin the follow coroutine when enabled.
             AutoFollowAtDistance = autoFollowAtDistance;
         }
+        
+        private void OnDisable() {
+            autoFollowDistanceCheck = null;
```

Comment:
@keveleigh `AutoFollowAtDistance` is a bool property and can't be set to `null`; did you mean `false`? I don't think that's the right answer. First, I don't think it would solve the issue that my PR solves, because the setter code exits early if the component is not enabled, and it would not nullify the coroutine. Second, `AutoFollowAtDistance` is a _setting_, and settings should not change when a component is disabled and re-enabled.

Reply:
Good call! Appreciate the thorough explanation and fully agreed.